.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: pymongo-auth-aws
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ["2.7", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10", "pypy-2.7", "pypy-3.7"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: install requests
+        run: |
+          python -m pip install requests==2.12	
+        if: ${{ matrix.python-version == 3.4 || matrix.python-version == 3.5}} 
+      - name: Test with python
+        run: |
+          python -m pip install botocore
```

Comment:
How about `python -m  pip install -e .`? This will automatically install all the dependencies.

Reply:
Done.